string axis    #one of surge, sway, heave, roll, pitch, yaw
bool en
---
bool success   #whether or not update succeeded
